### DISCUSSION TITLE
Netlify Identity from NPM instead of CDN OR Custom Widget

### DISCUSSION THREAD

**Discussion by hrishikesh-k** [StaticJsCMS/static-cms#811] [Q&A]
Hello!

I'm trying to use Netlify Identity Widget from NPM instead of CDN: https://github.com/netlify/netlify-identity-widget#module-api. It looks like Static CMS cannot detect this, unless I set it to `window.netlifyIdentity = NetlifyIdentityWidget`. Would it be possible to avoid setting this in the global scope by passing something like a config option to CMS?

Also, would it be possible to use my custom-built widget instead of the one provided by Netlify?

**Comment by KaneFreeman** [comment]
Moving this to an issue.
